packages/git.rb

```diff
@@ -3,379 +3,104 @@
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/git/2.33.1-musl_armv7l/git-2.33.1-musl-chromeos-armv7l.tpxz',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/git/2.33.1-musl_armv7l/git-2.33.1-musl-chromeos-armv7l.tpxz',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/git/2.33.1-musl_i686/git-2.33.1-musl-chromeos-i686.tpxz',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/git/2.33.1-musl_x86_64/git-2.33.1-musl-chromeos-x86_64.tpxz'
   })
   binary_sha256({
-    aarch64: 'f9dce49267eab2dd52233ff8bfdc6899f482dd17dc67a50eebbd2e531a2aa6b4',
-     armv7l: 'f9dce49267eab2dd52233ff8bfdc6899f482dd17dc67a50eebbd2e531a2aa6b4',
-       i686: '4e7ea06987c4ccea0ec900bffaba2184b48d3ca0d4d33d08134f23e1dc590129',
-     x86_64: '17eac279771387b441bd0175cd30451e5ac6f5cb2cd21f73f99477fadba5d947'
+    aarch64: '18052f1abdfd44c1386bbc89bd31f83b0952e5bfa479ed657ad4623f7f0349c6',
+     armv7l: '18052f1abdfd44c1386bbc89bd31f83b0952e5bfa479ed657ad4623f7f0349c6',
+       i686: 'e3de97e12f7177d8ee31dd250f92499c566f2bcda50e18f3a380e3b2fb5b9beb',
+     x86_64: '2e87ccba75d78143ee2d53ccb513f0a2a6d62604ae25288b71ed2aff5428aaa7'
   })
 
   depends_on 'ca_certificates' => :build
-  depends_on 'hashpipe' => :build
-  depends_on 'py3_pip' => :build
+  depends_on 'curl' => :build
   depends_on 'rust' => :build
-  depends_on 'valgrind' => :build
-
-  @deppath = "#{`pwd`.chomp}/deproot"
+  depends_on 'musl_brotli' => :build
+  depends_on 'musl_c_ares' => :build
+  depends_on 'musl_cyrus_sasl' => :build
+  depends_on 'musl_expat' => :build
+  depends_on 'musl_krb5' => :build
+  depends_on 'musl_libidn2' => :build
+  depends_on 'musl_libnghttp2' => :build
+  depends_on 'musl_libssh' => :build
+  depends_on 'musl_libunistring' => :build
+  depends_on 'musl_native_toolchain' => :build
+  depends_on 'musl_ncurses' => :build
+  depends_on 'musl_openssl' => :build
+  depends_on 'musl_zlib' => :build
+  depends_on 'musl_zstd' => :build
+
+  def self.patch
+    # Patch to prevent error function conflict with libidn2
+    # By replacing all calls to error with git_error.
+    system "sed -i 's,^#undef error\$,#undef git_error,' usage.c"
+    sedcmd = 's/\([[:blank:]]\)error(/\1git_error(/'.dump
+    system "grep -rl '[[:space:]]error(' . | xargs sed -i #{sedcmd}"
+    sedcmd2 = 's/\([[:blank:]]\)error (/\1git_error (/'.dump
+    system "grep -rl '[[:space:]]error (' . | xargs sed -i #{sedcmd2}"
+    system "grep -rl ' !!error(' . | xargs sed -i 's/ \!\!error(/ \!\!git_error(/g'"
+    system "sed -i 's/#define git_error(...) (error(__VA_ARGS__), const_error())/#define git_error(...) (git_error(__VA_ARGS__), const_error())/' git-compat-util.h"
+  end
 
   def self.build
-
-    # Create a new buildroot just for musl.
-    FileUtils.mkdir_p 'build'
-    FileUtils.mkdir_p 'deproot/include'
-    FileUtils.mkdir_p 'deproot/lib'
-    Dir.chdir 'deproot' do
```

Comment:
Change to
```
FileUtils.ln_s 'lib', 'deproot/lib64'
```

Reply:
This whole section has been removed so no need to change anything. :)

Reply:
😳